$DragonFly: src/gnu/usr.bin/cc34/cc_prep/patches/genconditions.c.patch,v 1.2 2004/12/21 13:10:48 joerg Exp $

===================================================================
RCS file: /home/joerg/wd/repository/dragonflybsd/src/contrib/gcc-3.4/gcc/genconditions.c,v
retrieving revision 1.1
diff -u -r1.1 genconditions.c
--- genconditions.c	20 Jun 2004 02:14:49 -0000	1.1
+++ genconditions.c	20 Dec 2004 20:48:11 -0000
@@ -125,11 +125,7 @@
    array initializers, fall back to assuming that all conditions\n\
    potentially vary at run time.  It works in 3.0.1 and later; 3.0\n\
    only when not optimizing.  */\n\
-#if (GCC_VERSION >= 3001) || ((GCC_VERSION == 3000) && !__OPTIMIZE__)\n\
-# define MAYBE_EVAL(expr) (__builtin_constant_p(expr) ? (int) (expr) : -1)\n\
-#else\n\
-# define MAYBE_EVAL(expr) -1\n\
-#endif\n");
+# define MAYBE_EVAL(expr) -1\n");
 }
 
 /* Write out one entry in the conditions table, using the data pointed
